app/containers/App/selectors.js

```diff
@@ -111,7 +116,20 @@ const makeEntitySelector = () => createSelector(
 
 const makeEntityMapSelector = () => createSelector(
```

Comment:
Agreed, will add an `makeEntityExtendedMapSelector` and revert the `makeEntityMapSelector` for the pure entities